cobalt/site/docs/development/setup-linux.md

```diff
@@ -199,3 +201,222 @@ to debug application performance, but also a great way to debug issues and
+mandatory since 2024 requirement, Cobalt should be executed in Evergreen mode.
+
+## Download and configure the official Google-built Cobalt binaries from GitHub
+1. Create output directory for evergreen
+
+    - Create the directory with arguments that meet your need
+      - Build type, ex: gold, qa
+      - Starboard API version, ex: 15, 16
+    - An example to create directory with build_type=qa and sb_api_version=16
+      ```sh
+      # go to cobalt root folder
+      export COBALT_SRC=${PWD}
+      export PYTHONPATH=${PWD}:${PYTHONPATH}
+
+      # setup evergreen build arguments
+      export EG_PLATFORM=evergreen-x64
+      export EG_BUILD_TYPE=qa
+      export SB_API_VER=16
+      export EVERGREEN_DIR=out/${EG_PLATFORM}_${EG_BUILD_TYPE}
+
+      gn gen $EVERGREEN_DIR --args="target_platform=\"$EG_PLATFORM\" use_asan=false build_type=\"$EG_BUILD_TYPE\" sb_api_version=$SB_API_VER"
+      ```
+
+1. Select Google-prebuilt Cobalt binaries from [GitHub](https://github.com/youtube/cobalt/releases)
+
+    - Choose the correct evergreen version based on the target device specification
+
+      Please note that the selected prebuilt binary must meet the settings used to create evergreen directory in previous step.
+      - Cobalt version you checked out, ex: 25.lts.1+
+      - Build type, ex: gold, qa
+      - Starboard API version, ex: 15, 16
+    - For Cobalt 25 with starboard API version 16, you need to use compressed version, ex:
+      - [`25.lts.1 release`](https://github.com/youtube/cobalt/releases/tag/25.lts.1)
+      - `Gold version`: cobalt_evergreen_5.1.2_x64_sbversion-16_release_compressed_20240629001855.crx
+
+      - `QA version`: cobalt_evergreen_5.1.2_x64_sbversion-16_qa_compressed_20240629001855.crx
+    - Right click the file and copy file URL
+
+1. Download and unzip the file
+
+    ```sh
+    export LOCAL_CRX_DIR=/tmp/cobalt_dl
+    rm -rf $LOCAL_CRX_DIR
+    mkdir -p $LOCAL_CRX_DIR
+
+    # paste prebuilt library URL and Download it to /tmp
+    # Please update URL according to your need
+    COBALT_CRX_URL=https://github.com/youtube/cobalt/releases/download/25.lts.1/cobalt_evergreen_5.1.2_x64_sbversion-16_qa_compressed_20240629001855.crx
+
+    wget $COBALT_CRX_URL -O $LOCAL_CRX_DIR/cobalt_prebuilt.crx
+
+    # Unzip the downloaded CRX file
+    unzip $LOCAL_CRX_DIR/cobalt_prebuilt.crx -d $LOCAL_CRX_DIR/cobalt_prebuilt
+    ```
+
+1. Copy the files to the appropriate directories
+
+    ```sh
+    cd $COBALT_SRC
+    mkdir -p $EVERGREEN_DIR/install/lib
+    cp -f $LOCAL_CRX_DIR/cobalt_prebuilt/lib/* $EVERGREEN_DIR/
+    cp -f $LOCAL_CRX_DIR/cobalt_prebuilt/lib/* $EVERGREEN_DIR/install/lib
+    cp -f $LOCAL_CRX_DIR/cobalt_prebuilt/manifest.json $EVERGREEN_DIR/
+    cp -rf $LOCAL_CRX_DIR/cobalt_prebuilt/content $EVERGREEN_DIR/
+    ```
+
+## Build executables running on target platform
+
+The Cobalt library is parpared in above steps. Next step is building an
```

Comment:
> executable accompany with StarBoard implementation to load the library.

I recommend not describing things too much here and instead just pointing readers to https://github.com/youtube/cobalt/blob/main/starboard/doc/evergreen/cobalt_evergreen_overview.md.

How about something like this?
"The next step is to build the partner-built Loader App and Crashpad Handler executables (described in https://github.com/youtube/cobalt/blob/main/starboard/doc/evergreen/cobalt_evergreen_overview.md)".

Reply:
I do refer the cobalt_evergreen_overview.md to modify the setup_linux.md file. But I need to modify the platform, copy contents from evergreen output folder and move crashpad_handler location for x64_x11 platform to work. In my opinion, it is good to have a full example that the user can follow and run by just copying and pasting the commands. I am going to mention evergreen document here but keeping the build steps for X64 here.